docs/source/getting-started/index.rst

```diff
@@ -8,59 +8,36 @@ Getting started with MythX security tools
 
 This page will show how to start using MythX in the shortest number of steps.
 
-Quickstart
-----------
-
-The quickest way to see MythX in action is by using a MythX tool such as the :ref:`MythX for Truffle <tools.truffle>` plugin.
-
-No registration is required for this, though the results will be limited. See the section on :ref:`using MythX for Truffle <tools.truffle>` for more details.
-
-#. In a terminal, navigate to the root of your local Truffle project.
-
-#. Install the MythX plugin:
-
-     .. code-block:: console
-
-        npm install truffle-security
-
-#. Analyze all the contracts in the project with MythX:
-
-     .. code-block:: console
-
-        truffle run verify
-
-This will run a security analysis in Trial Mode, which limits the returned results to three vulnerabilities only. For a more complete report, you will need to create an account with MythX.
-
-
 Creating an account
 -------------------
 
-To get the full functionality of MythX, you need to create an account.
+To use MythX, you need to create an account.
 
-.. note:: MythX requires `MetaMask <https://metamask.io>`_, a browser-based wallet application, in order to create accounts. You must be logged in to MetaMask before continuing. The active address will be associated with your account and be used for any plans you may purchase.
-
-#. Go to https://mythx.io and slick the :guilabel:`Sign Up` button on the top right.
+#. Go to https://mythx.io and click the :guilabel:`Sign Up` button on the top right.
 
 #. Fill out the registration form. You will need to supply a valid email address.
 
-   .. image:: img/registration.png
-
+   .. image:: img/registration2.png
+    :width: 50%
+    
 #. When finished, click :guilabel:`Complete Registration`.
 
-#. A MetaMask popup will display, asking for confirmation. Click :guilabel:`Sign` to continue.
+#. A success message will show, indicating your account has been created successfully. You will be sent an email to verify your email address. 
 
-   .. image:: img/metamasksignup.png
+   .. image:: img/success.png
       :width: 50%
 
-#. An API key will be generated for you and displayed. Please copy this key down, as you will not be able to retrieve it later. (You can generate a new one in your account dashboard if necessary.)
+.. note:: It is recommended that you link your Ethereum account to your profile as you will use this credential along with your password to access the MythX Api via client tools such as Remix.
 
-   .. image:: img/apikey.png
+   .. image:: img/metamask1.png
       :width: 50%
-
-#. You will be sent an email to verify your address. You will need to verify your email address before you can use the MythX service.
+      
+   .. image:: img/metamasksignup.png
+    :width: 50%
 
    .. Verify this
 
+
```

Comment:
Suggest taking out "This means that you can receive the complete report of vulnerabilities when running scans." because it's no longer a differentiation with the now-defunct Trial mode.